core/src/main/resources/reference.conf

```diff
@@ -59,11 +59,34 @@ akka {
 
     # Note that without further configuration, all other actors run on the default dispatcher
   }
+
+  coordinated-shutdown.phases {
+    # This phase is used to give time to Cromwell to abort all workflows upon shutdown. It's only used if system.abort-jobs-on-terminate = true
+    abort-all-workflows {
+      # /!\ This depends-on field should NOT be modified
+      depends-on = [service-unbind]
```

Comment:
I think they pretty much have to already for it to work (plus, the `akka` block should be there, and w/o the dispatchers it will fail).

Hold on a bit though, in between practice runs of this talk I'm trying to see what other people do.